Makefile

```diff
@@ -3,6 +3,7 @@ SOURCES := $(shell find $(SOURCEDIR) -name '*.go')
 ROOT := $(shell pwd)
 LOCAL_ENI_PLUGIN_BINARY=bin/plugins/eni
 LOCAL_IPAM_PLUGIN_BINARY=bin/plugins/ipam
+LOCAL_BRIDGE_PLUGIN_BINARY=bin/plugins/bridge
 GIT_PORCELAIN=$(shell git status --porcelain | wc -l)
 GIT_SHORT_HASH=$(shell git rev-parse --short HEAD)
 VERSION=$(shell cat $(ROOT)/VERSION)
```

Comment:
I think the argument was that we'll increment this version for changes to any of the plugins since we release all of them together (because they are part of the same repo). I think it's much simpler from a maintenance aspect to keep all of them at the same version rather than managing dependencies between them. If you have better suggestions to do this, please add them here and we can adapt to that model